app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
+}
+
+.nav-bar-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.1em;
+    text-decoration: none;
+    height: 76px;
+    display: block;
+    padding: 8px 12px 0 12px;
+}
+
+.nav-bar-link:hover {
+  background-color: rgba(0,0,0,.4);
 }
 
-#username {
+.username-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.45em;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered color, display, font-size, height, padding, text-decoration